src/Core/RetryingDispatchers/ExponentialRetryingDispatcher.cs

```diff
@@ -0,0 +1,44 @@
+﻿// <copyright file="ExponentialRetryingDispatcher.cs" company="Cognisant">
+// Copyright (c) Cognisant. All rights reserved.
+// </copyright>
+
+namespace CR.MessageDispatch.Core
+{
+    using System;
+
+    /// <summary>
+    /// Sets up a dispatcher that exponentially retries.
+    /// </summary>
+    /// <typeparam name="TMessage">Message Type</typeparam>
+    public class ExponentialRetryingDispatcher<TMessage> : RetryingDispatcher<TMessage>
+    {
+        private readonly TimeSpan _retryPeriod;
+        private readonly double _exponentialMultiplier;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ExponentialRetryingDispatcher{TMessage}"/> class.
+        /// </summary>
+        /// <param name="dispatcher">Dispatcher.</param>
```

Comment:
Should provide more detail on lots of these.

Reply:
I'm going to stop writing these everywhere, seems like all the docs could do with a review / rewording with more descriptive text.